packages/eslint-plugin-boxel/lib/rules/template-missing-invokable.js

```diff
@@ -0,0 +1,102 @@
+/** @type {import('eslint').Rule.RuleModule} */
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description:
+        'disallow missing helpers, modifiers, or components in \\<template\\> with auto-fix to import them',
+      category: 'Ember Octane',
+      url: 'https://github.com/cardstack/boxel/packages/eslint-plugin-boxel/tree/master/docs/rules/template-missing-invokable.md',
+      recommended: true,
+    },
+    fixable: 'code',
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          invokables: {
+            type: 'object',
+            additionalProperties: {
+              type: 'array',
+              prefixItems: [
+                {
+                  type: 'string',
+                  description: 'The name to import from the module',
+                },
+                { type: 'string', description: 'The module to import from' },
+              ],
+            },
+          },
+        },
+      },
+    ],
+    messages: {
+      'missing-invokable':
+        'Not in scope. Did you forget to import this? Auto-fix may be configured.',
+    },
+  },
+
+  create: (context) => {
+    const sourceCode = context.sourceCode;
+
+    // takes a node with a `.path` property
+    function checkInvokable(node) {
+      if (
+        node.path.type === 'GlimmerPathExpression' &&
+        node.path.tail.length === 0
+      ) {
+        if (!isBound(node.path.head, sourceCode.getScope(node.path))) {
+          const matched = context.options[0]?.invokables?.[node.path.head.name];
+          if (matched) {
```

Comment:
Add comment about how errors are not reported if there is no config match / fix.